<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TilesetEditDialog</class>
 <widget class="QDialog" name="TilesetEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1117</width>
    <height>706</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_TilesetID">
         <property name="text">
          <string>Tileset ID:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_ImportTileGraphic">
         <property name="text">
          <string>Import static tiles graphic from file</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_Tile8x8_ID">
         <property name="text">
          <string>Selected Tile8x8 Id:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGraphicsView" name="graphicsView_Tile8x8Editor">
         <property name="maximumSize">
          <size>
           <width>240</width>
           <height>240</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_palette">
         <property name="text">
          <string>Palette:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="horizontalSlider">
         <property name="maximum">
          <number>15</number>
         </property>
         <property name="pageStep">
          <number>1</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGraphicsView" name="graphicsView_paletteBar">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_Tile8x8SetGraphic">
         <property name="text">
          <string>Tile8x8 set:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGraphicsView" name="graphicsView_TilesetAllTile8x8"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_Tile16SetGraphic">
         <property name="text">
          <string>Tile16 set:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGraphicsView" name="graphicsView_TilesetAllTile16"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_SelectedTile16Id">
         <property name="text">
          <string>Seleted Tile16 Id:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_EventId">
         <property name="text">
          <string>Event Id:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_EventId"/>
       </item>
       <item>
        <widget class="QLabel" name="label_TerraiId">
         <property name="text">
          <string>Terrain Id:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_TerrainId"/>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_Components">
         <property name="title">
          <string>Components:</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <item row="3" column="3">
             <widget class="QCheckBox" name="checkBox_BottomLeftHFlip">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_tileposition">
              <property name="text">
               <string>position</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="label_tile8x8Id">
              <property name="text">
               <string>tile8x8 id</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="label_paletteId">
              <property name="text">
               <string>palette id</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QSpinBox" name="spinBox_TopLeftTileId"/>
            </item>
            <item row="0" column="3">
             <widget class="QLabel" name="label_HFlip">
              <property name="text">
               <string>H Flip</string>
              </property>
             </widget>
            </item>
            <item row="0" column="4">
             <widget class="QLabel" name="label_VFlip">
              <property name="text">
               <string>V Flip</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_TopLeft">
              <property name="text">
               <string>top left</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QSpinBox" name="spinBox_TopLeftpaletteId"/>
            </item>
            <item row="1" column="3">
             <widget class="QCheckBox" name="checkBox_TopLeftHFlip">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QSpinBox" name="spinBox_TopRightpaletteId"/>
            </item>
            <item row="2" column="1">
             <widget class="QSpinBox" name="spinBox_TopRightTileId"/>
            </item>
            <item row="3" column="1">
             <widget class="QSpinBox" name="spinBox_BottomLeftTIleId"/>
            </item>
            <item row="3" column="2">
             <widget class="QSpinBox" name="spinBox_BottomLeftpaletteId"/>
            </item>
            <item row="1" column="4">
             <widget class="QCheckBox" name="checkBox_TopLeftVFlip">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="2" column="3">
             <widget class="QCheckBox" name="checkBox_TopRightHFlip">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="2" column="4">
             <widget class="QCheckBox" name="checkBox_TopRightVFlip">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_TopRight">
              <property name="text">
               <string>top right</string>
              </property>
             </widget>
            </item>
            <item row="3" column="4">
             <widget class="QCheckBox" name="checkBox_BottomLeftVFlip">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_BottomLeft">
              <property name="text">
               <string>bottom left</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_BottomRight">
              <property name="text">
               <string>bottom right</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QSpinBox" name="spinBox_BottomRightTileId"/>
            </item>
            <item row="4" column="2">
             <widget class="QSpinBox" name="spinBox_BottomRightpaletteId"/>
            </item>
            <item row="4" column="3">
             <widget class="QCheckBox" name="checkBox_BottomRightHFlip">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="4" column="4">
             <widget class="QCheckBox" name="checkBox_BottomRightVFlip">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TilesetEditDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TilesetEditDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
